### DISCUSSION TITLE
How to make Custom Events?

### DISCUSSION THREAD

**Discussion by Outspending** [Minestom/Minestom#1912] [Help] [answered]
I have a forked version of minestom-ce and I was wondering how I can call, and listen for custom events. If someone could help thanks so much.

**Comment by bebranator** [comment]
i think no custom packets but events yeeees!!!!

**Comment by Codestech1** [comment, marked as answer]
Why custom packets? If you make server with ability to join from vanilla clients they won't work.
For custom events just create class implementing `Event` interface and call them with `EventDispatcher#call`. You can listen to them like for normal minestom event.

**Reply by Outspending** [reply]
Yeah i didn't mean to say "Custom packets", i meant like how do i send packets whenever. Sorry for the miscommunication :)

**Reply by Codestech1** [reply]
Ah then just create a packet instance and send it via `Player#sendPacket`

**Reply by Outspending** [reply]
Ah okay, thanks so much 🥰
